pkg/network/ebpf/c/tracer/stats.h

```diff
@@ -21,6 +21,11 @@ static __always_inline __u64 offset_rtt_var();
 #endif
 
 static __always_inline conn_stats_ts_t *get_conn_stats(conn_tuple_t *t, struct sock *sk) {
+    conn_stats_ts_t *cst = bpf_map_lookup_elem(&conn_stats, t);
+    if (cst) {
```

Comment:
Is this to try and save the call to `get_sk_cookie`?

Reply:
Yes, and a call to `bpf_map_update_elem` since that is also causing a lot of errors in bpf telemetry.

Reply:
`BPF_EXIST` shouldn't really be treated like an error, we should remedy that.

Reply:
@usamasaqib